fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -368,9 +370,22 @@ private void generateNextTaskRun() {
     }
 
     private void refreshExternalTable(TaskRunContext context) {
-        for (Pair<BaseTableInfo, Table> tablePair : snapshotBaseTables.values()) {
-            BaseTableInfo baseTableInfo = tablePair.first;
-            Table table = tablePair.second;
```

Comment:
Good catch! So need check mv's base table infos if its base table is external table everywhere, because external tables maybe dropped in any time?